### DISCUSSION TITLE
Twitch Log-In doesn't work with Thorium.

### DISCUSSION THREAD

**Discussion by DadSaysImCool** [Alex313031/Thorium-Win#102] [Ideas]
<img width="362" alt="thorium_jCsiiFbuxX" src="https://github.com/Alex313031/Thorium-Win/assets/90170980/68ccdb10-fd48-4f7a-a04b-bf6aa641481a">

Is there any way to make Twitch think I am still on a supported Broswer, like Chrome?

Thank you.

**Comment by StephanBruh** [comment]
Works fine for me

**Reply by DadSaysImCool** [reply]
Well, do you have any idea why it doesn't for me, and what I could try so it does?

**Reply by gz83** [reply]
Please try the latest version 138. If you still have problems, please let us know.
